mobile-app/app/screens/AppNavigator/screens/Loans/components/LoanCardsV2.tsx

```diff
@@ -86,57 +110,109 @@ export function LoanCardsV2(props: LoanCardsProps): JSX.Element {
   ) as LoanVaultActive;
 
   return (
-    /* Known intermittent issue wherein the two-column layout is not followed in web - FlashList */
-    <ThemedFlashList
-      estimatedItemSize={116}
-      contentContainerStyle={tailwind("pt-4 pb-2")}
-      data={filteredLoanTokens}
-      /* This tells FlashList to rerender if any of the props below is updated */
-      extraData={{
-        isVaultReady,
-        activeVault,
-      }}
+    <ThemedScrollViewV2
+      contentContainerStyle={tailwind("px-5 py-8 w-full")}
       ref={ref}
-      numColumns={2}
-      ListEmptyComponent={
-        <View style={tailwind("mt-1")}>
-          <SkeletonLoader row={6} screen={SkeletonLoaderScreen.LoanV2} />
-        </View>
-      }
-      renderItem={({
-        item,
-        index,
-      }: {
-        item: LoanToken;
-        index: number;
-      }): JSX.Element => {
-        return (
-          <View style={{ flexBasis: "98%" }}>
-            <LoanCard
-              symbol={item.token.symbol}
-              displaySymbol={item.token.displaySymbol}
-              interestRate={item.interest}
-              price={item.activePrice}
-              loanTokenId={item.tokenId}
-              onPress={() => {
-                navigation.navigate({
-                  name: "BorrowLoanTokenScreen",
-                  params: {
-                    loanToken: item,
-                    vault: activeVault,
-                  },
-                  merge: true,
-                });
-              }}
-              testID={`loan_card_${index}`}
-              isBorrowHidden={!isVaultReady}
-            />
+    >
+      <SearchInputV2
+        ref={searchRef}
+        value={searchString}
+        showClearButton={searchString !== ""}
+        placeholder={translate(
+          "screens/LoansScreen",
+          "Search available loan tokens"
+        )}
+        containerStyle={tailwind("flex-1", [
```

Comment:
we cannot put the padding px-5 directly on the parent since the LoanCard have additional padding

```suggestion
        containerStyle={tailwind("flex-1 mx-5", [
```